src/main/resources/kibana-rest.yml

```
@@ -109,6 +118,7 @@ kibana_fleet_settings:
 kibana_reporting_diagnose_browser:
   versions:
     ">= 8.16.0": "/internal/reporting/diagnose/browser"
+    # ">= 9.0.0": "uri [/internal/reporting/diagnose/browser] with method [get] exists but is not available with the current configuration"
```

Does this mean we should add `< 9.0.0` to the above line to disable it?

```suggestion
    ">= 8.16.0 < 9.0.0": "/internal/reporting/diagnose/browser"
    # ">= 9.0.0": "uri [/internal/reporting/diagnose/browser] with method [get] exists but is not available with the current configuration"
```